docs/guides/advanced/ServerRendering.md

```diff
@@ -53,7 +53,7 @@ render(<Router history={history} routes={routes} />, mountNode)
 You need to do
 
 ```js
-match({ routes, location }, (error, redirectLocation, renderProps) => {
-  render(<Router {...renderProps} history={history} />, mountNode)
+match({ history, routes }, (error, redirectLocation, renderProps) => {
```

Comment:
On the server, as above, you still `match({ routes, location })` and render a `<RouterContext>`.

On the client, here, you `match({ history, routes })` and render a `<Router>`. Since `history` is going to be some browser history, it doesn't need to be told its location.